<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo68</class>
 <widget class="QDialog" name="dlgDemo68">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lectura y Estadistica de un Archivo Grande con QThread</string>
  </property>
  <widget class="QLabel" name="lblArchivo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona el Archivo de Texto con la Data a Analizar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtArchivo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>731</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSeleccionarArchivo">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>40</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProcesar">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Procesar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstResultado">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>861</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbrArchivo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>851</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblRegistros">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>560</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total de Registros Procesados:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotal">
   <property name="geometry">
    <rect>
     <x>744</x>
     <y>560</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
